src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -234,24 +238,24 @@ public void Dispose()
             StringCollection results = null;
             switch(metadata.MetadataTypeName)
             {
-                case ("Table"):
-                    results = scripter.GetTableScripts(metadata.Name, metadata.Schema);
-                    break;
-                case ("View"):
-                    results = scripter.GetViewScripts(metadata.Name, metadata.Schema);
-                    break;
-                case("StoredProcedure"):
-                    results = scripter.GetStoredProcedureScripts(metadata.Name, metadata.Schema);
-                    break;
-                case("Schema"):
-                    results = scripter.GetSchemaScripts(metadata.Name, metadata.Schema);
-                    break;
-                case("Database"):
-                    results = scripter.GetDatabaseScripts(metadata.Name, metadata.Schema);
-                    break;
-                default:
-                    results = null;
-                    break;
+                // case ("Table"):
+                //     results = scripter.GetTableScripts(metadata.Name, metadata.Schema, null);
+                //     break;
```

Comment:
dead code? please either remove, reenable, or leave a comment why the code is being disabled with the date and your alias in the comment.